tests/functional/z_cluster/rag/test_olsconfig_with_odfragimage.py

```
@@ -0,0 +1,266 @@
+                        )
+            else:
+                results.append(entry)
+
+            if qtype == "invalid":
+                if "data foundation" in ans1.lower():
+                    failures.append(
+                        f"Q{qid}: Hallucinated answer (unexpected data foundation mention)"
+                    )
+
+            elif qtype == "no_rag_answer":
+                if not is_uncertain(ans1):
```

Ruff (0.15.15)

[warning] Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)